### ISSUE TITLE
Something goes wrong

### ISSUE THREAD

**Issue by PT2727** [LaTeX-Bibitem-Styler/latex-bibitemstyler#8]
I am using the LaTex-BibitemStyler (https://github.com/LaTeX-Bibitem-Styler/latex-bibitemstyler) to help me order my references in order of appearance. I am using this since I do not understand how BibTex or Biblatex or these work. I am a beginner in efficiently putting references in LaTex and I found this LaTex-BibitemStyler code for Python which could help me.  

I am using this as an input on python 3.7.6 on Pyzo on Windows:
>>>styler = Styler("C:\\OrderingLiterature\\main_file.tex", "C:\\OrderingLiterature\\bibliography_original.tex", "C:\\OrderingLiterature\\bibliography_ordered.tex", 2)
>>>styler.StyleBibitems()

However, I get a comment:
"Please enter a valid main Tex file path"

Why isn't the one I use valid?

**Comment by oorasch** [comment]
If you run the py-file w/o command line arguments it says:

Please run the program with 4 arguments:
    - The main tex file path (use double '\\' if on Windows)
    - The bibliography Tex file name
    - The output bibliography Tex file name
    - The bibliography style:
0 - PLAIN (Original order)
1 - ALPHA (Alphanumerical order)
2 - UNSRT (Cite order of appearance)

Try using \\\ instead of \\?

**Comment by PT2727** [comment]
Oh yes sorry I am an idiot. That is indeed exactly what they say in the command line.

For maybe people with a similar problem.
I also found that 

"C:\\OrderingLiterature\\main_file.txt" also doesn't work, instead just use the main file path. "main_file.txt" as the code puts the current directory in it already. 
I also used .txt instead of .tex and that apparently also works.

Thank you very much for your help
